### PR TITLE
Repo revamp inital commit

### PR DESCRIPTION
**Pull request by anraghun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,119 +1,61 @@
- # Arc Enabled Kubernetes Validation Program 
+ # Azure Arc Validation Program 
 
- Azure Arc aims at bringing management and services to any infrastructure. Deploy and manage applications on Kubernetes clusters everywhere using GitOps.  
+While Azure Arc can be deployed on any K8s clusters, the Arc validation program provides distinct visibility for integrated K8s platforms and promises quality, support and consistency for Azure Arc customers across different hybrid clusters.
```
Comment (shashankbarsin) [inline README.md]: 'promises' is going too far as we can never provide 100% guarantee, suggest rephrasing this sentence

File: README.md [hunk 2]
```diff
@@ -1,119 +1,61 @@
- # Arc Enabled Kubernetes Validation Program 
+ # Azure Arc Validation Program 
 
- Azure Arc aims at bringing management and services to any infrastructure. Deploy and manage applications on Kubernetes clusters everywhere using GitOps.  
+While Azure Arc can be deployed on any K8s clusters, the Arc validation program provides distinct visibility for integrated K8s platforms and promises quality, support and consistency for Azure Arc customers across different hybrid clusters. 
 
-While Azure Arc can be deployed on any K8s clusters, validation provides distinct visibility for integrated K8s platforms and promises quality, support and consistency for Azure Arc customers across hybrid clusters. 
+To realize the promise of Arc feature consistency and high quality on partner platforms, we have built the conformance test framework and automation to continuously test and validate the Arc functionality on validated platforms.
```
Comment (shashankbarsin) [inline README.md]: same as above

File: README.md (hunk 1 above)
Comment (anraghun) [reply README.md]: Addressed this

File: README.md (hunk 2 above)
Comment (anraghun) [reply README.md]: Addressed this

**Pull request merged**